docs/how-to-guides/software-management/software-support.md

```diff
@@ -1,34 +1,52 @@
 # Software Support
-This document provides information about the software support process, including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
 
+!!! Note "How to Submit Issues" 
+    Our code is managed [via GitHub](https://github.com/DOI-USGS)
 
-## Step 1: Scheduling the Support Sprint
+    Any issues with our software portfolio should be tracked in our repos through [GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues). Links for some our bigger repos: 
+
+    * [ISIS](https://github.com/DOI-USGS/ISIS3/issues)
+    * [ALE](https://github.com/DOI-USGS/ale/issues)
+    * [USGSCSM](https://github.com/DOI-USGS/usgscsm/issues)
+
+    This document provides information about the software support process. That is, how our development team goes about addressing these issues. Including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
```

Comment:
These sentences do not make sense here. This is introductory material. Move to the top of this doc.